persistence/session/redis/store.go

```diff
@@ -111,6 +111,7 @@ func (s *Store) Iterate(fn session.IterateFn) error {
 		}
 		if len(arr) >= 1 {
 			for _, v := range arr[1:] {
+				if len( v.([]interface{})) > 1 {
```

Comment:
It is not necessary to add the if statement.  If the length is 0, the `for range` part just skip it. 
Why do you want to add this statement？ Did you find some bugs?
BTW, it is recommended to run `go fmt` to format your code before making a PR